docs/admin/extensible-admission-controllers.md

```diff
@@ -20,11 +21,107 @@ the following:
 * They need to be compiled into kube-apiserver.
 * They are only configurable when the apiserver starts up.
 
-1.7 introduces two alpha features, *Initializers* and *External Admission
-Webhooks*, that address these limitations. These features allow admission
-controllers to be developed out-of-tree and configured at runtime.
+Two features, *Admission Webhooks* (beta in 1.9) and *Initializers*, 
```

Comment:
I think we need to mention that Initializers are alpha, because the old line described both as alpha and the new line only says the state of admission webhooks, but not initializers. Maybe something like:
```Two features, *Admission Webhooks* (beta in 1.9) and *Initializers* (alpha), ```